Formula/lib/libvpx.rb

```diff
@@ -51,3 +54,47 @@ def install
     system "ar", "-x", "#{lib}/libvpx.a"
   end
 end
+
+__END__
+diff --git a/build/make/configure.sh b/build/make/configure.sh
+index cc5bf6ce40..9c449492b0 100644
+--- a/build/make/configure.sh
++++ b/build/make/configure.sh
+@@ -832,7 +832,7 @@ process_common_toolchain() {
+         tgt_isa=x86_64
+         tgt_os=`echo $gcctarget | sed 's/.*\(darwin1[0-9]\).*/\1/'`
+         ;;
+-      *darwin2[0-4]*)
++      *2[0-5]*)
```

Comment:
Was removing `darwin` intentional here?

Reply:
good catch, that seems an incorrect update.

Reply:
Whoops, no 😅